src/countOccurrences.js

```diff
@@ -18,7 +18,11 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  return phrase
+    .toLowerCase()
+    .split('')
+    .filter(elementOfPhrase => letter === elementOfPhrase)
```

Comment:
не всегда нужно усложнять :)
```suggestion
    .filter(element => letter === element)
```